### PR TITLE
Windows compatibility

### PR DESCRIPTION
**Pull request by damieng**
Tested on Windows and Mac (one spec fails on the Mac but was failing before this PR)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Do not convert line endings as is directly compared with in spec
+spec/fixtures/saved-html.html -text
```
Comment (50Wliu) [inline .gitattributes]: atom/atom's .gitattributes does this a bit differently: https://github.com/atom/atom/blob/master/.gitattributes#L1

File: lib/main.coffee [hunk 2]
```diff
@@ -34,22 +34,18 @@ module.exports =
     atom.commands.add '.tree-view .file .name[data-name$=\\.txt]', 'markdown-preview:preview-file', previewFile
 
     atom.workspace.addOpener (uriToOpen) =>
-      try
-        {protocol, host, pathname} = url.parse(uriToOpen)
-      catch error
-        return
-
-      return unless protocol is 'markdown-preview:'
+      [ protocol, path ] = uriToOpen.split('://')
```
Comment (as-cii) [inline lib/main.coffee]: 🔥 Space before and after square brackets.

File: .gitattributes (hunk 1 above)
Comment (as-cii) [inline .gitattributes]: That's interesting, @50Wliu. @damieng: should we use that approach here too?

**Comment by as-cii** [comment]
Two small comments, otherwise looks good. 👍

File: .gitattributes (hunk 1 above)
Comment (damieng) [inline .gitattributes]: I could go either way. This tells it to leave the file as-is - the other pattern forces specific line endings.

Comment (as-cii) [inline .gitattributes]: Ok, I am 🆒 keeping it this way then. 👍

**Pull request merged**